### ISSUE TITLE
Missing Predis

### ISSUE THREAD

**Issue by edgrosvenor** [writingink/wink#202]
Upon logging in (Laravel 6.0), I am presented with: Class 'Predis\Client' not found

Obviously, installing `composer require predis/predis` solves it, but should it be a requirement in composer since it's apparently required by the package? I remember reading something about some changes / deprecations around redis packages in Laravel core so is this maybe an unfinished transition situation?

Anyway, I'd normally just submit a pull request to add the requirement to composer but I wasn't sure how you'd want to handle it.

**Comment by tvbeek** [comment]
Isn't that depending on your config? Is is listed in the upgrade guide from Laravel: https://laravel.com/docs/6.x/upgrade#redis-default-client

I think it shouldn't be a requirements for this project. (In fact I use it without redis so shouldn't need it.)

**Comment by edgrosvenor** [comment]
Cool. Thanks. Yes, that's where I read it. And you're absolutely right. The weird thing is that this was a pretty freshly installed Laravel 6 installation, not upgraded from a previous version. And I never touched the database config file because I don't even use a database on this one. So at the moment when I installed Laravel 6, the redis client hadn't been changed from predis in the config yet. I just did a totally fresh installation and it's phpredis now, so all is well.
